string file_path
string file_name
---
bool result
